userguide/content/en/docs/Theme installation/Docsy-As-Module/start-from-scratch.md

```diff
@@ -20,8 +20,7 @@ At your command prompt, run the following:
 hugo new site my-new-site
 cd  my-new-site
 hugo mod init github.com/me/my-new-site
-hugo mod get github.com/google/docsy@0.2.0-pre
-hugo mod get github.com/google/docsy/dependencies@0.2.0-pre
+hugo mod get github.com/google/docsy@v0.2.0
```

Comment:
- Oh! So we only need one Hugo module now?
- I thought that we needed two so as to ensure that we didn't break things for projects that aren't switching to Hugo modules (yet)? Cf. #848.

Reply:
> * Oh! So we only need one Hugo module now?

No. We still need the second module `docsy/dependencies`.
But there is no need to bring it in manually, since the module is pulled in automatically from `go.mod` at root level..

> * I thought that we needed two so as to ensure that we didn't break things
>  for projects that aren't switching to Hugo modules (yet)?

Yes, that's still valid. We need two modules, the second one brings in `dependencies/config.toml` which actuallys mounts the dependencies BS + FA at their correct places in the project hierarchy.